client/libDisk.cc

```diff
@@ -120,6 +127,15 @@ int initLibDisk(char* address, char* token) {
     return LIBDISK_WEBSOCKET_BAD_DATA;
   }
 
+  switch (registration_response.responsecode()) {
+    case tinyfs::ClientRegistrationResponse_ResponseCode_SUCCESS:
+      break;
+    case tinyfs::ClientRegistrationResponse_ResponseCode_BAD_TOKEN:
+      LOGERR("Token was rejected by server");
```

Comment:
in the event that a student sees this, we may want to be more helpful. Something like "Token rejected by the server -- please contact lab TA or Professor."